test/providers/store/file.js

```diff
@@ -105,28 +105,15 @@ const data = [
   }
 ]
 
-describe('path to coordinates mapping', () => {
```

Comment:
we are no longer mapping path to coordinates, we are reading explicit coordinate metadata